openpdf/src/main/java/com/lowagie/text/ImageLoader.java

```diff
@@ -160,6 +160,18 @@ public static Image getBmpImage(byte[] imageData) {
         }
     }
 
+    /**
+     * Creates an Image from an array of tiff image bytes.
+     * For JRE < 9, `ImageIO.read()` requires a supporting library in the classpath to read tiffs.
+     * Options are:
```

Comment:
On Java8 without jai-imageio-core in the classpath, `ImageIO.read()` throws a NullPointerException